yash-datalake/nested/blobTriggerdFunction.json

```diff
@@ -112,7 +112,7 @@
     {
       "type": "Microsoft.Web/serverfarms",
       "name": "[parameters('serverfarms_name')]",
-      "apiVersion": "2019-08-01",
+      "apiVersion": "2018-11-01",

```

Comment:
same as above, why are we rolling back the apiVersion

Reply:
This API version was creating problem while running spark job in data factory. Tested all possible API version came out to the conclusion that Storage account and azure functions need to have similar API versions.
That's I roll backed it to the version where both are same